libs/CMakeLists.txt

```diff
@@ -96,7 +97,7 @@ if(HPX_WITH_TESTS)
   endif()
 
   if (HPX_WITH_TESTS_HEADERS)
-    add_hpx_pseudo_target(tests.headers.modules)
+    add_custom_target(tests.headers.modules)
```

Comment:
Please don't use `add_custom_target` directly, use `add_hpx_pseudo_target` instead. Otherwise this will break Windows builds.

Reply:
Grr, thanks. I'll make a separate PR to have `create_library_skeleton.py` fixed in master finally.

Reply:
https://github.com/STEllAR-GROUP/hpx/commit/b3503fc5a4aac7c1bf618993d9e0f726194439d3. I pushed directly to master, and will fix this PR shortly as well.

Reply:
Thanks! I was planning to change that a while back but forgot as well.